src/System.Diagnostics.DiagnosticSource/tests/DiagnosticSourceEventSourceBridgeTests.cs

```diff
@@ -13,6 +13,22 @@ namespace System.Diagnostics.Tests
     //Complex types are not supported on EventSource for .NET 4.5
     public class DiagnosticSourceEventSourceBridgeTests : RemoteExecutorTestBase
     {
+        // For reasons I am not sure of, we run all these tests in their own sub-process using RemoteInvoke()
```

Comment:
> For reasons I am not sure of

It's to avoid issues when tests run in parallel, e.g. where one DiagnosticSource test ends up causing events that get picked up by another test evaluating the bridge.